README.md

```diff
@@ -31,9 +31,10 @@ appropriate one:
 Limitations
 ------------
 
-This plugin cannot be used for retiring accounts in the [Open edX E-Commerce Service](https://github.com/openedx/ecommerce).[^ecom]
+This plugin cannot be used for retiring accounts in the [Open edX E-Commerce Service](https://github.com/openedx/ecommerce),[^ecom] nor the [Course Discovery Service](https://github.com/openedx/course-discovery).[^discovery].
 
-[^ecom]: See [Issue #36](https://github.com/hastexo/tutor-contrib-retirement/issues/36) in this repository for a related discussion of missing functionality in E-Commerce account retirement.
+[^ecom]: See [Issue #36](https://github.com/hastexo/tutor-contrib-retirement/issues/36) for background.
+[^discovery]: See [Issue #37](https://github.com/hastexo/tutor-contrib-retirement/issues/36) for background.
```

Comment:
The link possibly has a typo: Discovery issue is described in issue #39, not in the #36 (the e-commerce issue).

And the link name is `Issue #37` but it leads to #36 if I understand it correctly.

```suggestion
[^discovery]: See [Issue #39](https://github.com/hastexo/tutor-contrib-retirement/issues/39) for background.
```

Reply:
Ah yes, good catch, thanks! Fixed in 2d3d23399f126b3c0cf17b1c1102787a456bbcc6.